admin-portal-frontend/src/app/components/VerticalNavBar.tsx

```diff
@@ -26,6 +27,24 @@ import SearchComponent from './SearchComponent';
         );
       }
 
+      interface Category {
+        _id: string;
+        title: String;
+        items: [];
+        type: String;
+      }
+      
+      const [items, setItems] = useState<Category[]>([]);
+      useEffect(() => {
+        (async () => {
+          const categories = await getAllCategories();
+          setItems(categories);
+        })();
+      }, []);
+
+      const generalPrinciples = items.filter(category => category.type === 'General Principle');
```

Comment:
Make sure there is a tab called 'All' at the top: 
<img width="213" alt="image" src="https://github.com/TritonSE/DFM-Sideline-Sidekick-App/assets/76824702/3644ba92-fef7-4992-8444-6c227c051968">